code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -66,1037 +66,1649 @@ Follow by example and make good judgement based on length which list to include
+		gender = MALE
+		length = 4
+		chatname = "big afro"
 
-	longeralt2
-		name = "Long Hair Alt"
-		icon_state = "hair_longeralt2"
-		length = 2
-		chatname = "Long Hair"
+	amanita
+		name = "Amanita"
+		icon_state = "hair_amanita"
+		chatname = "short curls"
 
-	longest
-		name = "Very Long Hair"
-		icon_state = "hair_longest"
+	amanita_long
+		name = "Amanita, Long"
+		icon_state = "hair_amanita_long"
 		length = 3
-		chatname = "Very Long Hair"
+		chatname = "long curls"
 
-	longfringe
-		name = "Long Fringe"
-		icon_state = "hair_longfringe"
-		length = 2
-		chatname = "Long Fringed Hair"
+	amanita_long_alt
+		name = "Amanita, Long Alt"
+		icon_state = "hair_amanita_long_alt"
+		length = 3
+		chatname = "long curls"
 
-	longeralt
-		name = "Longer Fringe"
-		icon_state = "hair_vlongfringe"
+	amazon
+		name = "Amazon"
+		icon_state = "hair_amazon"
+		gender = FEMALE
 		length = 2
-		chatname = "Long Fringed Hair"
+		chatname = "long hair"
 
-	halfbang
-		name = "Half-banged Hair"
-		icon_state = "hair_halfbang"
-		length = 2
-		chatname = "Half-banged Hair"
+	averagejoe
+		name = "Average Joe"
+		icon_state = "hair_averagejoe"
+		gender = MALE
+		chatname = "short hair"
 
-	halfbangalt
-		name = "Half-banged Hair Alt"
-		icon_state = "hair_halfbang_alt"
-		length = 2
-		chatname = "Half-banged Hair"
+	baldingfade
+		name = "Balding Fade"
+		icon_state = "hair_baldingfade"
+		gender = MALE
+		length = 0
+		chatname = "bald head"
 
-	ponytail1
-		name = "Ponytail 1"
-		icon_state = "hair_ponytail"
-		length = 2
-		chatname = "Ponytail"
+	baldinghair
+		name = "Balding Hair"
+		icon_state = "hair_baldinghair" //hair_e
+		gender = MALE // turnoff!
+		length = 0
+		chatname = "balding hair"
 
-	ponytail2
-		name = "Ponytail 2"
-		icon_state = "hair_pa"
+	bangs
+		name = "Bangs"
+		icon_state = "hair_bangs"
 		gender = FEMALE
 		length = 2
-		chatname = "Ponytail"
-
-	ponytail3
-		name = "Ponytail 3"
-		icon_state = "hair_ponytail3"
-		length = 2
-		chatname = "Ponytail"
+		chatname = "fringe"
 
-	ponytail4
-		name = "Ponytail 4"
-		icon_state = "hair_ponytail4"
+	bangs_short
+		name = "Bangs, Short"
+		icon_state = "hair_bangs_short"
 		gender = FEMALE
-		length = 2
-		chatname = "Ponytail"
+		chatname = "fringe"
 
-	ponytail5
-		name = "Ponytail 5"
-		icon_state = "hair_ponytail5"
-		length = 2
-		chatname = "Ponytail"
+	bangs_veryshort
+		name = "Bangs, Very Short"
+		icon_state = "hair_bangs_veryshort"
+		chatname = "fringe"
 
-	ponytail6
-		name = "Ponytail 6"
-		icon_state = "hair_ponytail6"
-		gender = FEMALE
-		length = 2
-		chatname = "Ponytail"
+	bedhead
+		name = "Bedhead"
+		icon_state = "hair_bedhead"
+		chatname = "messy locks"
 
-	ponytail7
-		name = "Ponytail 7"
-		icon_state = "hair_ponytail7"
-		gender = FEMALE
-		length = 2
-		chatname = "Ponytail"
+	bedhead2
+		name = "Bedhead 2"
+		icon_state = "hair_bedhead2"
+		chatname = "messy locks"
 
-	sideponytail
-		name = "Side Ponytail"
-		icon_state = "hair_stail"
+	bedhead3

```

Comment:
By the way. This was "removed" because I resprited and renamed it. Why are you bringing this back?

Reply:
Firstly,
Thank you for adding a new sprite. I take it you also did "Neat Long"? I'm presuming it's "Neat" but I do want to apologise first because I actually didn't even realise that they were the same hairstyles.
But...
Maybe that starts to tie in with what I'm about to say.

Secondly,
Hear me out a little.

I know it's two pixels removed and one pixel moved (or thereabouts) but I feel like even such a subtle change can make a huge difference. That doesn't just apply with the icons presented here, but art as a whole, as well as hairstyles in real life.

_now i'm going to evoke my innate feng shui mastery here_ and note that these slight changes can alter the frame of the face, and it may very well be evident whether people are conscious of it or not.

On a tacked-on note, I also want to speak based not just from my own experiences in the past or my recent experiences having to trawl through icon files, but from what I observe of other people in which they have slight changes they'd like to see with the hairstyle they're using. It's still generally recognisable as XYZ hairstyle, but there's some minor variance to it which alters its outward appearance.
Despite how it looks, I didn't add every single little variety of hairstyle I found; I figured it would probably not really fly too well with many devs here (correct me if I'm wrong). I did find some that added a little more "plump" to the tops of some hairstyles. In real life, maybe it might be subtle as its change, but when we're talking 32x32 pixels, it can and in these cases does alter the proportions of an appearance.
But I do want to note that "Bedhead 3" and "Neat" seem, afaik, to be the only hairstyles that have the same sort of shape. I get that we can't cover every single little thing (or can we?), but if a player wishes for this or that then at the very least we have that end covered for their sake. Going back to what I said before about framing of the face, I do want to note that "Bedhead 3" has considerable fringe compared to "Neat". That can matter to quite a number of people I have on my mind.

Reply:
If I'm going to be really idealistic and presumptuous for a moment here, just for hypothesis: do you think Bedhead 3 and/or Neat should be renamed so that they are together? And if so, which one(s) and to what?

Reply:
No, we shouldn't cover the same thing, because it leads to the hairstyle list becoming unnavigable. Let Bedhead 3 die.